{"name": "test_testcase2", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "def test_testcase2():\n        print(\"running the test case2\")\n>       assert False\nE       assert False\n\ntestLoader.py:10: AssertionError"}, "attachments": [{"name": "stdout", "source": "2ed25a1d-6f42-4357-907a-22640aac91a7-attachment.txt", "type": "text/plain"}], "start": 1587984318358, "stop": 1587984318358, "uuid": "7c123ad5-f645-4bce-83ce-a00af68094f4", "historyId": "953dbbac3632a86a8962e99edb0c895e", "testCaseId": "925302bfa09516062cc7b946b82feeb9", "fullName": "testLoader#test_testcase2", "labels": [{"name": "suite", "value": "testLoader"}, {"name": "host", "value": "DESKTOP-3A2H5GC"}, {"name": "thread", "value": "7372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testLoader"}]}